lib/active_admin/filters/forms.rb

```diff
@@ -31,7 +31,7 @@ def default_input_type(method, options = {})
           case column.type
           when :date, :datetime
             :date_range
-          when :string, :text
+          when :string, :text, :citext
```

Comment:
I'd be ok with just this since it's meant to behave no different a string.

I don't want or have the time to deal with introducing Postgres to the test suite.

Reply:
Ok, I will test one thing this weekend, I suspect I could define a custom citext type (it doesn't exist since there is no postgres). If that works, I should be able to test with sqlite, even if behind the scenes is just varchar

Reply:
@Fire-Dragon-DoL are you able to patch this change and test it in your setup? I use Postgres but haven't used citext before. I'm ok with accepting just this change and no tests. It should work fine as is with Postgres since its text based like the other two.

Reply:
I'll update the pr to remove tests tomorrow morning. Sorry for the long wait

Reply:
If more postgres specialization do arise, I should probably work on a way to introduce postgres to a subset of the test suite